### ISSUE TITLE
Hi-C data

### ISSUE THREAD

**Issue by ValWood** [pombase/website#1479]
We have a triage section for Hi_C data, but we can't host it yet.

This  paper PMID:27548313 is a HI-C dataset, with some curation, so I don't want to classify as Hi-C
We can resolve this when we can triage as "curatable" and something else
In the meantime, we need a list. This is probably not the best place but I'll sort that once I have some time.

We have tickets related to this on various trackers

**Comment by ValWood** [comment]
This is a placeholder ticket, nothing to do yet

**Comment by ValWood** [comment]
"3C" type of analysis, showing physical clustering of meiotic DSB hotspots by ligation, immunoprecipitation, and deep sequencing
Fowler et al., PNAS 2018 (Figure 4B and C, and S7A and B)
(this should probably move to a page where we collect 3C data).
PMID: 30217891
(https://github.com/pombase/website/files/3371680/Fowler.et.al.PNAS.2018.complete.pdf)

**Comment by kimrutherford** [comment]
> We have a triage section for Hi_C data, but we can't host it yet.

JBrowse v2 supports Hi-C tracks: https://jbrowse.org/jb2/docs/user_guides/hic_track/

V2 issue:
 - https://github.com/pombase/website/issues/2038

**Comment by ValWood** [comment]
This might be a nice new. feature for the next paper...

@PCarme  do you think this would be useful? I'm not sure how many HI-C datasets we have for fission yeast

**Comment by PCarme** [comment]
Yes, this could be useful. I don't know how many Hi-C papers there are in S. pombe, but I could look into it

**Comment by ValWood** [comment]
I just remembered!

https://curation.pombase.org/pombe/view/list/triage_3d_genome_organization?model=track

**Comment by kimrutherford** [comment]
> JBrowse v2 supports Hi-C tracks: https://jbrowse.org/jb2/docs/user_guides/hic_track/

The data needs to be in `.hic` format for JBrowse:
https://github.com/aidenlab/juicer/wiki/Data#hic-files

I had a look at PMID:27548313.  It says:

```
Data described in this paper were deposited into the National Center for Biotechnology Information
Sequence Read Archive database with accession number SRP061635.
```

So they're submitted the reads but there is no mention of depositing the processed data anywhere.

This software can process reads into `.hic` files: https://github.com/aidenlab/juicer
but I don't think we necessarily want to be doing that ourselves.

**Comment by ValWood** [comment]
OK @PCarme  could you check the HI-C data for .hic files.
If there is none, we can mail the authors for the processed files.
I'm sure they will comply if we say they will be announced as the firsr Hi-C tracks in JBrowse.

**Comment by ValWood** [comment]
Actually I will put out a call and we'll see who responds.

**Comment by kimrutherford** [comment]
Shall we close this for now?
